test.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
```

Comment:
If you want to keep your test page, a fun thing to do would be to turn it into a styleguide (http://styleguides.io/), since it includes so many examples of different elements already 😄